src/providers/swift.rs

```diff
@@ -0,0 +1,177 @@
+use super::Provider;
+use crate::nixpacks::{
+    app::App,
+    environment::Environment,
+    nix::pkg::Pkg,
+    phase::{BuildPhase, InstallPhase, SetupPhase, StartPhase},
+};
+use anyhow::{bail, Result};
+
+static DEFAULT_SWIFT_VERSION: &str = "5.5.3";
```

Comment:
It looks like this version isn't actually in `AVAILABLE_SWIFT_VERSIONS`...

Reply:
The current version on nixpkgs is 5.5.3, thus I see no need to add it there.